src/patterns/task-list-pages/index.md.njk

```diff
@@ -65,14 +65,52 @@ Where possible, allow users to complete tasks in any order. This will help them
 
 ## Research on this pattern
 
-Read a blog post about [testing and iterating the task list page pattern](https://designnotes.blog.gov.uk/2017/04/04/weve-published-the-task-list-pattern/).
+This pattern was originally developed and tested by a team at the Government Digital Service (GDS). 
 
-### <a name="next-steps"></a>Next steps
+The team built prototypes of task lists for 3 services, Register as a childminder, Learn to drive and Transport goods and tested them with 34 users over 5 rounds of research. 
+
+The pattern was iterated after each round of testing.
+
+You can read more about [testing and iterating the task list page pattern](https://designnotes.blog.gov.uk/2017/04/04/weve-published-the-task-list-pattern/).
+
+### Known issues and gaps
+
+There is evidence that the completed tag might be misleading to some users.
+
+In one instance, [a user saw a task marked ‘completed’ and thought this meant they had submitted their response](https://github.com/alphagov/govuk-design-system-backlog/issues/72#issuecomment-413159884). As a result, they didn’t continue to the ‘Check and submit your response’ task, and were later issued with a court order for not responding.
+
+In addition, user research and feedback on this pattern has shown that: 
+
+- some screen reader users are frustrated by having to tab through every section each time they return to the task list after completing a task
+- some services need users to complete tasks in a particular order, for example, a user must fill in an application before they can pay 
+once a few tasks have been completed it becomes harder to scan the page and spot incomplete tasks
```

Comment:
I thought this was just a hard to read bullet point but it looks like there is a missing bullet point for the line starting "once"